.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           name: Deploy to gcloud test project
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              ./ci/deploy.sh ui
+              ./ci/deploy_ui.sh
```

Comment:
This is a nice improvement. If you end up touching this again, I would challenge you to create a UI `circle-deploy` task modeled after the same in API, which would allow us to delete this file altogether.

Reply:
Sure - will do if I need to make any more substantial changes here.